packages/amplify-graphql-auth-transformer/src/utils/schema.ts

```diff
@@ -65,20 +66,20 @@ export const getModelConfig = (directive: DirectiveNode, typeName: string, isDat
       createdAt: 'createdAt',
       updatedAt: 'updatedAt',
     },
-  });
+  }, { deepMergeArguments: featureFlags.getBoolean('shouldDeepMergeDirectiveConfigDefaults', false) });
```

Comment:
It would be nice to have a utility function in the V2 core that generates defaults for these `GetArgumentsOptions` so that we can refactor this to something like `getDefaultArgumentsOptions(featureFlags)`. This utility method will be only used by V2 transformers and provides a way to centrally handle this. You can even specify the name of FF i.e `shouldDeepMergeDirectiveConfigDefaults` as a const there.

Reply:
Done